src/iceberg/catalog/rest/constant.h

```diff
@@ -28,17 +28,16 @@
 
 namespace iceberg::rest {
 
-constexpr std::string_view kHeaderContentType = "Content-Type";
-constexpr std::string_view kHeaderAccept = "Accept";
-constexpr std::string_view kHeaderXClientVersion = "X-Client-Version";
-constexpr std::string_view kHeaderUserAgent = "User-Agent";
+inline const std::string kHeaderContentType = "Content-Type";
```

Comment:
Why did you apply this change? I would prefer `constexpr std::string_view` instead of `const std::string`.